plugin-tests/composer.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "wordpress/performance",
+  "type": "wordpress-plugin",
+  "license": "GPL-2.0-or-later",
+  "description": "Performance plugin from the WordPress Performance Team, which is a collection of standalone performance modules.",
+  "homepage": "https://wordpress.org/plugins/performance-lab/",
+  "keywords": [
+    "performance",
+    "wordpress"
+  ],
+  "support": {
+    "issues": "https://github.com/WordPress/performance/issues"
+  },
+  "require-dev": {
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.7",
+    "phpcompatibility/php-compatibility": "^9.3",
+    "phpunit/phpunit": "^4|^5|^6|^7|^8|^9",
```

Comment:
Fun fact: PHPUnit doesn't need to be required if we're using `yoast/phpunit-polyfills`, because it [requires PHPUnit as a dependency itself](https://github.com/Yoast/PHPUnit-Polyfills/blob/1.0.4/composer.json#L27) and handles proper versioning for you.

Reply:
@joemcgill have removed, thanks. We should probably consider this rule for the main plugin, as that is where I borrowed the file from.

Reply:
Agreed! It's some thing that we could check for in all of our plug-ins that use the polyfills library